### ISSUE TITLE
Display Keyboard Shortcuts in the Settings

### ISSUE THREAD

**Issue by MattiaWasFound** [apankrat/nullboard#105]
I can't quite figure out what the shortcuts are. I looked on the git page and your website, but I can't find them there.

Short of looking at the source code, I don't know how someone's supposed to figure them out :)

It might be useful to have them appear in the settings menu, or when an information button is clicked?

On this note, is there a shortcut for fast creation of notes? 

Thanks!

**Comment by dustinsterk** [comment]
I agree this would be helpful, I ended up adding this to my personal version to remind myself:

 [Tab] = Cycles through notes
 [Control + Enter] = New note below
 [Control + Shift + Enter] = Note above
 [Control + *] = • (Windows)
 [Command + Shift + 8] = • (MacOS)   -- ONLY WORKS WITH MINOR CODE TWEAK (see Open PRs)
 [Alt/Option + R] = Toggle raw mode
 [Alt/Option + UpArrow] = Collapse
 [Alt/Option + DownArrow] = Expand
 [Control] or [CapsLock] = Show links (allow them to be clicked)
